test/e2e/resource_quota.go

```diff
@@ -147,6 +148,84 @@ var _ = framework.KubeDescribe("ResourceQuota", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
+	It("should create a ResourceQuota and capture the life of an uninitialized pod.", func() {
```

Comment:
I leave the test here to show what the end goal is. I can move the test to #50344 if that's better.